src/App/Header/UserMenu/UserMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { Authenticator, useAuthenticator } from "@aws-amplify/ui-react";
+import { Popover } from "@headlessui/react";
+import Icon from "components/Icon";
+import Menu from "./Menu";
+
+export default function UserMenu() {
+  const { user, signOut, route } = useAuthenticator((context) => [
+    context.route,
+  ]);
+
+  return (
+    <Popover className="relative">
+      {route === "authenticated" ? (
+        <Popover.Button className="cursor-pointer contents">
+          <Icon type="User" size={24} className="text-white" />
+        </Popover.Button>
+      ) : (
+        <Popover.Button className="btn-orange px-3 h-10 rounded-lg text-sm">
+          Sign in
```

Comment:
"Verify email" panel colors are off in "dark mode", text should be white
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/f580f852-3609-4547-b51e-8c40df62991b)

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2427/commits/44b41584072149fb4d20d23b7b9f565ece2abc23